packages/ui5-application-writer/src/types.ts

```diff
@@ -79,6 +79,11 @@ export interface AppOptions {
      * Excludes the index.html from the template and does not add the `start-noflp` script in package.json
      */
     generateIndex?: boolean;
+
+    /**
+     * Enables specific ui5 libs for FPM apps
+     */
+    loadFpmUi5Libs?: boolean;
```

Comment:
Sorry, no. You are in a generic UI5 writer and you add a property that is Fiori elements specific. Please change this to e.g. allow defining additional libs but don't put FE/FPM specific logic in the generic ui5 writer